internal/services/cosmos/cosmosdb_account_resource.go

```diff
@@ -463,6 +476,58 @@ func resourceCosmosDbAccount() *pluginsdk.Resource {
 
 			"cors_rule": common.SchemaCorsRule(),
 
+			"restore": {
+				Type:     pluginsdk.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &pluginsdk.Resource{
+					Schema: map[string]*pluginsdk.Schema{
+						"restore_source": {
```

Comment:
can we remove restore (as its part of the block name) and be more clear on what this expects??
```suggestion
						"source_cosmosdb_account|database_id": {
```

Reply:
Updated